browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1278,17 +1278,32 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_IMPORT_BOOKMARKS_SELECT_FILE" desc="Import Bookmarks file chooser text">
         Select a File to Import bookmarks
       </message>
-      <message name="IDS_IMPORT_BOOKMARKS_SUCCESS" desc="Import Bookmarks success text message.">
+      <message name="IDS_EXPORT_BOOKMARKS_SUCCESS_TITLE" desc="Export Bookmarks success title text.">
+        Export successful
```

Comment:
Suggestion: `Export succeeded` (to match the opposite `Export failed`).

Reply:
@anthonypkeane wdyt?